test/Microsoft.Azure.WebJobs.Host.FunctionalTests/TestDoubles/MemoryBlobStore.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Azure.WebJobs.Host.FunctionalTests.TestDoubles
     internal class MemoryBlobStore
     {
         private readonly ConcurrentDictionary<string, Container> _items = new ConcurrentDictionary<string, Container>();
-        private ServiceProperties _properties = new ServiceProperties();
+        private ServiceProperties _properties = new ServiceProperties(new LoggingProperties(), new MetricsProperties(), new MetricsProperties(), new CorsProperties());
```

Comment:
@mathewc I wanted your opinion on this -- if you look at https://github.com/Azure/azure-webjobs-sdk/blob/b1fb20b0edf0530d62127d6dd9f50cc9345ca7b6/src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/BlobLogListener.cs#L184, you'll see that we do no null check on `serviceProperties.Logging`. In the previous Storage SDK, this was always initialized so we never hit it.

Should we change the behavior here to create a `new LoggingProperties()` object it if it's null on the `serviceProperties`? Or just add a null check and skip the logging setup? I'm not sure whether you'd ever actually get a null Logging property or what happens if that logging setup block is skipped.